server/app/views/v1/host_nodes/health.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+json.id @node.node_id
+json.name @node.name
+json.etcd @etcd_health
```

Comment:
Yes, etcd key is good but the content of that object is somewhat confusing to me.